lib/utils/room_status_extension.dart

```diff
@@ -43,8 +43,9 @@ extension RoomStatusExtension on Room {
       }
       return L10n.of(context)!.onlineLongTimeAgo;
     }
-    return L10n.of(context)!
-        .countParticipants(summary.mJoinedMemberCount.toString());
+
+    return L10n.of(context)!.countParticipants(
+      ((summary.mInvitedMemberCount ?? 0) + (summary.mJoinedMemberCount ?? 0)).toString());
```

Comment:
when you invite new member, did this count is updated ?

Reply:
yes
![yes](https://github.com/linagora/twake-on-matrix/assets/48354990/b82e330c-f698-440f-9c13-769191a21b44)